yaml_test.go

```diff
@@ -271,6 +271,10 @@ func TestJSONToYAML(t *testing.T) {
 			`{"t":null}`,
 			"t: null\n",
 			nil,
+		},{
```

Comment:
add a test case that exercises the 2.2.8 and default 2.4.0 indent behavior to make sure we remain compatible by default